utils/generate-module-api-doc.rb

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env ruby
-
 # coding: utf-8
```

Comment:
`# coding: utf-8` needs to be the first line, so no blank line before this. (My tools complained.)